docs/content/快速上手/build-real-time-data-lake-tutorial-zh.md

```diff
@@ -0,0 +1,284 @@
+
+![Flink UI](/_static/fig/real-time-data-lake-tutorial/flink-ui.png "Flink UI")
+
+### 准备数据
+1. 进入 MySQL 容器中
+    ```shell
+    docker-compose exec mysql mysql -uroot -p123456
+    ```
+2. 创建数据和表，并填充数据
+   
+   创建两个不同的数据库，并在每个数据库中创建两个表，作为 `user` 表分库分表下拆分出的表。
+   ```sql
+    CREATE DATABASE db_1;
+    USE db_1;
+    CREATE TABLE user_1 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+    INSERT INTO user_1 VALUES (110,"user_110","Shanghai","123567891234","user_110@foo.com");
+   
+    CREATE TABLE user_2 (
+      id INTEGER NOT NULL PRIMARY KEY,
+      name VARCHAR(255) NOT NULL DEFAULT 'flink',
+      address VARCHAR(1024),
+      phone_number VARCHAR(512),
+      email VARCHAR(255)
+    );
+   INSERT INTO user_2 VALUES (120,"user_120","Shanghai","123567891234","user_120@foo.com");
+   ```
+   ```sql
+   CREATE DATABASE db_2;
+   USE db_2;
+   CREATE TABLE user_1 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512),
+     email VARCHAR(255)
+   );
+   INSERT INTO user_1 VALUES (110,"user_110","Shanghai","123567891234", NULL);
+
+   CREATE TABLE user_2 (
+     id INTEGER NOT NULL PRIMARY KEY,
+     name VARCHAR(255) NOT NULL DEFAULT 'flink',
+     address VARCHAR(1024),
+     phone_number VARCHAR(512),
+     email VARCHAR(255)
+   );
+   INSERT INTO user_2 VALUES (220,"user_220","Shanghai","123567891234","user_220@foo.com");
+   ```
+
+## 在 Flink SQL CLI 中使用 Flink DDL 创建表
+首先，使用如下的命令进入 Flink SQL CLI 容器中：
+```shell
+docker-compose exec sql-client ./sql-client
+```
+我们可以看到如下界面：
+
+![Flink SQL Client](/_static/fig/real-time-data-lake-tutorial/flink-sql-client.png  "Flink SQL Client" )
+
+然后，进行如下步骤：
+1. 开启 checkpoint，每隔3秒做一次 checkpoint
+   
+   Checkpoint 默认是不开启的，我们需要开启 Checkpoint 来让 Iceberg 可以提交事务。
+   并且，mysql-cdc 在 binlog 读取阶段开始前，需要等待一个完整的 checkpoint 来避免 binlog 记录乱序的情况。
+   ```sql
+   -- Flink SQL                   
+   Flink SQL> SET execution.checkpointing.interval = 3s;
+   ```
+2. 创建 MySQL 分库分表 source 表
+   
+   创建 source 表 `user_source` 来捕获MySQL中所有 `user` 表的数据，在表的配置项 `database-name` , `table-name` 使用正则表达式来匹配这些表。 
+   并且，`user_source` 表也定义了 metadata 列来区分数据是来自哪个数据库和表。
+   ```sql
+   -- Flink SQL
+   Flink SQL> CREATE TABLE user_source (
+       database_name STRING METADATA VIRTUAL,
+       table_name STRING METADATA VIRTUAL,
+       `id` DECIMAL(20, 0) NOT NULL,
+       name STRING,
+       address STRING,
+       phone_number STRING,
+       email STRING,
+       PRIMARY KEY (`id`) NOT ENFORCED
+     ) WITH (
+       'connector' = 'mysql-cdc',
+       'hostname' = 'mysql',
+       'port' = '3306',
+       'username' = 'root',
+       'password' = '123456',
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

It appears your code contains a hardcoded password. Hardcoded secrets or credentials can allow attackers to bypass authentication methods and perform malicious actions. We recommend revoking access to resources using this credential and storing future credentials in a management service such as [AWS Secrets Manager](https://aws.amazon.com/secrets-manager/).

[Learn more about the use of hardcoded credentials](https://cwe.mitre.org/data/definitions/798.html)